woocommerce/wc-api-dev
# Only run if woocommerce is valid.

This code checks if woocommerce is active and at least the minimum
version before initializing.

This also removes the previous notice.

To Test:

1. Install `wc-api-dev` and ensure correct operation (Load products list in Store on WP)
2. In the wc-api-dev-class.php file, edit the WC_MIN_VERSION to be higher than your current WooCommerce version (I used 4.0.0)
3. The products list should no longer work, verifying that `wc-api-dev` is not installing api hooks.
